src/adapters/convex/products/staking/convexStakingAdapter.ts

```diff
@@ -319,7 +305,7 @@ export class ConvexStakingAdapter
       extraRewardTokenManagers,
     } = await this.fetchPoolMetadata(protocolTokenAddress)
 
-    const responsePromises = protocolRewardTokens?.map(
+    const responsePromises = protocolRewardTokens!.map(
```

Comment:
If this is an internal call (I can only see it being called within `LpStakingProtocolAdapter`), we probably want to make this property non-optional so that we don't have to assert.